lib/postcode_validation/use_case/address_list.rb

```diff
@@ -8,8 +8,8 @@ def initialize(address_list_gateway:, logger: nil)
         @logger = logger
       end
 
-      def execute(postcode:, country:)
-        addresses = matched_addresses(postcode, country)
+      def execute(postcode:, country:, more_results_id: nil)
```

Comment:
I'm confused by more_results_id. is this a PCA Anywhere concept? Would this make users of this use case aware of PCA Anywhere?